### ISSUE TITLE
Browse notes by directory structure

### ISSUE THREAD

**Issue by esm7** [redsolver/noteless#64]
Hi,
Noteless is nice and sleek, but for the purpose of interoperability with other Markdown editors (e.g. desktop apps like Obsidian), I'd love to have it center around a standard directory structure and be able to just read Markdown files with links and tags. The current structure just makes it very unnatural to use Noteless as the mobile front for an existing Markdown repo.

**Comment by redsolver** [comment]
You can currently already enable "Show virtual tags" in the settings to browse notes by subdirectory. As far as I understand you also want to create new notes in a subdirectory, right?

**Comment by esm7** [comment]
Yes, in essence what I want (and I'm sure many users will appreciate) is
interoperability with desktop Markdown editors, and most of these (at least
the standard ones) are based on a directory view.

**Comment by redsolver** [comment]
Noteless aims to be compatible with Notable (https://notable.app/) which uses a flat directory structure. Maybe I will add support for interoperability with other directory structures of other desktop apps in the future.

**Comment by redsolver** [comment]
I just published Version 1.4.6 (https://github.com/redsolver/noteless/releases/tag/v1.4.6) which contains a new experimental "Dendron" mode which can be enabled in the settings. This mode improves compatibility with other Markdown-based note-taking tools which save all Markdown files directly in one root folder together with attachments (images, ...) and have filenames which are different from the Markdown title. With this mode you can basically just select a folder in the settings and Noteless then loads all Markdown files in that folder. In addition, it shows the filename and path in the note list and editor.
